naga/tests/out/hlsl/atomicTexture-int64.hlsl

```diff
@@ -0,0 +1,21 @@
+struct NagaConstants {
+    int first_vertex;
+    int first_instance;
+    uint other;
+};
+ConstantBuffer<NagaConstants> _NagaConstants: register(b0, space1);
+
+RWTexture2D<uint> image : register(u0);
+
+int ZeroValueint() {
+    return (int)0;
+}
+
+[numthreads(2, 1, 1)]
+void cs_main(uint3 id : SV_GroupThreadID)
+{
+    image.InterlockedMax64(int2(0, 0),1uL);
```

Comment:
error: no member named 'InterlockedMax64' in 'RWTexture2D<unsigned int>'
im not sure how to get a `RWByteAddressBuffer` from a 'RWTexture2D<unsigned int>'
https://microsoft.github.io/DirectX-Specs/d3d/HLSL_SM_6_6_Int64_and_Float_Atomics.html#interlockedmax

Reply:
You don't, you use RWTexture2D like so: https://github.com/microsoft/DirectXShaderCompiler/blob/75ff50caa046a054747ae15b5c1910a4c8aa1917/tools/clang/test/HLSLFileCheck/hlsl/intrinsics/atomic/atomic.hlsl#L58

Reply:
This indeed fixed the hlsl!